content/doc/book/system-administration/administering-jenkins.adoc

```diff
@@ -0,0 +1,135 @@
+---
+layout: section
+title: Administering Jenkins
+---
+
+== `JENKINS_HOME` directory
```

Comment:
We think the "Jenkins Home Directory" section belongs in https://www.jenkins.io/doc/book/managing/system-configuration/ and that page can be switched to no longer be a work in progress.